[2025-01-17T18:15:37.528+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-17T18:15:37.611+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_second_dbt_dag.run_dbt_model scheduled__2025-01-15T00:00:00+00:00 [queued]>
[2025-01-17T18:15:37.677+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_second_dbt_dag.run_dbt_model scheduled__2025-01-15T00:00:00+00:00 [queued]>
[2025-01-17T18:15:37.681+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2025-01-17T18:15:37.768+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): run_dbt_model> on 2025-01-15 00:00:00+00:00
[2025-01-17T18:15:37.792+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_second_dbt_dag', 'run_dbt_model', 'scheduled__2025-01-15T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/my_second_dbt_dag.py', '--cfg-path', '/tmp/tmpqua40ixw']
[2025-01-17T18:15:37.796+0000] {standard_task_runner.py:105} INFO - Job 32: Subtask run_dbt_model
[2025-01-17T18:15:37.803+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=8303) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-17T18:15:37.805+0000] {standard_task_runner.py:72} INFO - Started process 8306 to run task
[2025-01-17T18:15:38.021+0000] {task_command.py:467} INFO - Running <TaskInstance: my_second_dbt_dag.run_dbt_model scheduled__2025-01-15T00:00:00+00:00 [running]> on host 10a7333bcdf9
[2025-01-17T18:15:38.201+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_second_dbt_dag' AIRFLOW_CTX_TASK_ID='run_dbt_model' AIRFLOW_CTX_EXECUTION_DATE='2025-01-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-15T00:00:00+00:00'
[2025-01-17T18:15:38.203+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-17T18:15:56.967+0000] {logging_mixin.py:190} INFO - 18:15:46  Running with dbt=1.6.1
18:15:46  Registered adapter: postgres=1.6.1
18:15:50  Found 4 models, 2 seeds, 2 sources, 0 exposures, 0 metrics, 464 macros, 0 groups, 0 semantic models
18:15:51  
18:15:51  Concurrency: 1 threads (target='dev')
18:15:51  
18:15:51  1 of 4 START sql table model jin_schema.my_first_dbt_model ..................... [RUN]
18:15:53  1 of 4 OK created sql table model jin_schema.my_first_dbt_model ................ [SELECT 2 in 1.68s]
18:15:53  2 of 4 START sql table model jin_schema.my_fifth_dag_dbt_model ................. [RUN]
18:15:54  2 of 4 OK created sql table model jin_schema.my_fifth_dag_dbt_model ............ [SELECT 2 in 0.87s]
18:15:54  3 of 4 START sql table model jin_schema.my_second_dbt_model .................... [RUN]
18:15:54  3 of 4 OK created sql table model jin_schema.my_second_dbt_model ............... [SELECT 1 in 0.27s]
18:15:54  4 of 4 START sql table model jin_schema.my_third_dbt_model ..................... [RUN]
18:15:54  4 of 4 OK created sql table model jin_schema.my_third_dbt_model ................ [SELECT 2 in 0.27s]
18:15:54  
18:15:54  Finished running 4 table models in 0 hours 0 minutes and 3.76 seconds (3.76s).
18:15:54  
18:15:54  Completed successfully
18:15:54  
18:15:54  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2025-01-17T18:15:56.972+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-17T18:15:57.008+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-17T18:15:57.013+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my_second_dbt_dag, task_id=run_dbt_model, run_id=scheduled__2025-01-15T00:00:00+00:00, execution_date=20250115T000000, start_date=20250117T181537, end_date=20250117T181557
[2025-01-17T18:15:57.110+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-17T18:15:57.155+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
